### PR TITLE
[de] Translate instructions for kubectl on Windows

### PR DESCRIPTION
**Pull request by termdew**
This Pull Request adds a new translation for the installation instructions for kubectl on Windows.
Furthermore the general kubectl installation article is removed as it's outdated and contains a few errors.

### CODE REVIEW

File: content/de/docs/tasks/tools/install-kubectl-windows.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+1. Stellen Sie sicher, dass die aktuellste Version installiert wurde:
+
+   ```powershell
+   kubectl version --client
+   ```
+
+1. Wechseln Sie in Ihr Benutzerverzeichnis:
+
+   ```powershell
+   # Falls Sie cmd.exe verwenden, führen Sie folgendes aus: cd %USERPROFILE%
```
Comment (bene2k1) [inline content/de/docs/tasks/tools/install-kubectl-windows.md:124]:
```suggestion
   # Falls du `cmd.exe` verwendest, führe folgendes aus: cd %USERPROFILE%
```

File: content/de/docs/tasks/tools/install-kubectl-windows.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+   mkdir .kube
+   ```
+
+1. Wechseln Sie in das das `.kube` Verzeichnis, das Sie soeben erstellt haben:
+
+   ```powershell
+   cd .kube
+   ```
+
+1. Konfigurieren Sie `kubectl`, um ein externes Kubernetes Cluster zu verwenden:
```
Comment (bene2k1) [inline content/de/docs/tasks/tools/install-kubectl-windows.md:140]:
```suggestion
1. Konfiguriere `kubectl`, um einen externen Kubernetes Cluster zu verwenden:
```

File: content/de/docs/tasks/tools/install-kubectl-windows.md [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+
+## Konfiguration von kubectl überprüfen
+
+{{< include "included/verify-kubectl.md" >}}
+
+## Optionale Konfigurationen und Plugins für kubectl
+
+## Auto-Vervollständigung aktivieren
+`kubectl` unterstützt Auto-Vervollständigung in Bash, Zsh, Fish und PowerShell.
+Somit können Sie sich viel Schreibarbeit sparen.
```
Comment (bene2k1) [inline content/de/docs/tasks/tools/install-kubectl-windows.md:158]:
```suggestion
Somit kannst du dir viel Schreibarbeit sparen.
```

File: content/de/docs/tasks/tools/install-kubectl-windows.md [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+
+Im Folgenden finden Sie eine Anleitung, um die Auto-Vervollständigung für PowerShell einzurichten.
```
Comment (bene2k1) [inline content/de/docs/tasks/tools/install-kubectl-windows.md:160]:
```suggestion
Im Folgenden findest du eine Anleitung, um die Auto-Vervollständigung für PowerShell einzurichten.
```

File: content/de/docs/tasks/tools/install-kubectl-windows.md [hunk 5]
```diff
@@ -0,0 +1,207 @@
...
+
+### Das `kubectl convert`-Plugin installieren
+1. Laden Sie die aktuellste Version mit folgendem Kommando herunter:
```
Comment (bene2k1) [inline content/de/docs/tasks/tools/install-kubectl-windows.md:163]:
```suggestion
1. Lade die aktuellste Version mit folgendem Kommando herunter:
```

File: content/de/docs/tasks/tools/install-kubectl-windows.md [hunk 6]
```diff
@@ -0,0 +1,207 @@
...
+   ```powershell
+   curl.exe -LO "https://dl.k8s.io/release/v{{< skew currentPatchVersion >}}/bin/windows/amd64/kubectl-convert.exe"
+   ```
+
+1. Überprüfen Sie die Binärdatei (optional):
```
Comment (bene2k1) [inline content/de/docs/tasks/tools/install-kubectl-windows.md:168]:
```suggestion
1. Überprüfe die Binärdatei (optional):
```

File: content/de/docs/tasks/tools/install-kubectl-windows.md [hunk 7]
```diff
@@ -0,0 +1,207 @@
...
+
+   Laden Sie die Prüfsumme von `kubectl-convert` herunter:
```
Comment (bene2k1) [inline content/de/docs/tasks/tools/install-kubectl-windows.md:170]:
```suggestion
   Lade die Prüfsumme von `kubectl-convert` herunter:
```

File: content/de/docs/tasks/tools/install-kubectl-windows.md [hunk 8]
```diff
@@ -0,0 +1,207 @@
...
+   ```powershell
+   curl.exe -LO "https://dl.k8s.io/v{{< skew currentPatchVersion >}}/bin/windows/amd64/kubectl-convert.exe.sha256"
+   ```
+
+   Validieren Sie die `kubectl-convert`-Binärdatei mit der heruntergeladenen Prüfsumme:
```
Comment (bene2k1) [inline content/de/docs/tasks/tools/install-kubectl-windows.md:175]:
```suggestion
   Validiere die `kubectl-convert`-Binärdatei mit der heruntergeladenen Prüfsumme:
```

File: content/de/docs/tasks/tools/install-kubectl-windows.md [hunk 9]
```diff
@@ -0,0 +1,207 @@
...
+
+   - Vergleichen Sie das Ergebnis von `CertUtil` manuell mit der heruntergeladenen Prüfsumme:
```
Comment (bene2k1) [inline content/de/docs/tasks/tools/install-kubectl-windows.md:177]:
```suggestion
   - Vergleiche das Ergebnis von `CertUtil` manuell mit der heruntergeladenen Prüfsumme:
```

File: content/de/docs/tasks/tools/install-kubectl-windows.md [hunk 10]
```diff
@@ -0,0 +1,207 @@
...
+
+     ```cmd
+     CertUtil -hashfile kubectl-convert.exe SHA256
+     type kubectl-convert.exe.sha256
+     ```
+
+   - Verwenden Sie den `-eq`-Operator von PowerShell, um den Hash der Binärdatei mit der Prüfsumme zu vergleichen:
```
Comment (bene2k1) [inline content/de/docs/tasks/tools/install-kubectl-windows.md:184]:
```suggestion
   - Verwende den `-eq`-Operator von PowerShell, um den Hash der Binärdatei mit der Prüfsumme zu vergleichen:
```

File: content/de/docs/tasks/tools/install-kubectl-windows.md [hunk 11]
```diff
@@ -0,0 +1,207 @@
...
+
+     ```powershell
+     $($(CertUtil -hashfile .\kubectl-convert.exe SHA256)[1] -replace " ", "") -eq $(type .\kubectl-convert.exe.sha256)
+     ```
+
+1. Fügen Sie den Ordner mit der `kubectl-convert` Binärdatei zu Ihrer `PATH`-Umgebungsvariable hinzu.
```
Comment (bene2k1) [inline content/de/docs/tasks/tools/install-kubectl-windows.md:190]:
```suggestion
1. Füge den Ordner mit der `kubectl-convert` Binärdatei zu deiner `PATH`-Umgebungsvariable hinzu.
```

File: content/de/docs/tasks/tools/install-kubectl-windows.md [hunk 12]
```diff
@@ -0,0 +1,207 @@
...
+
+1. Stellen Sie sicher, dass das Plugin korrekt installiert wurde:
```
Comment (bene2k1) [inline content/de/docs/tasks/tools/install-kubectl-windows.md:192]:
```suggestion
1. Stelle sicher, dass das Plugin korrekt installiert wurde:
```

File: content/de/docs/tasks/tools/install-kubectl-windows.md [hunk 13]
```diff
@@ -0,0 +1,207 @@
...
+
+   ```shell
+   kubectl convert --help
+   ```
+
+   Wenn kein Fehler ausgegeben wurde, wurde das Plugin erfolgreich installiert.
+
+1. Entfernen Sie die Installationsdateien nach der Installation des Plugins:
```
Comment (bene2k1) [inline content/de/docs/tasks/tools/install-kubectl-windows.md:200]:
```suggestion
1. Entferne die Installationsdateien nach der Installation des Plugins:
```

**Review by bene2k1 — CHANGES_REQUESTED** [review]

**Comment by termdew** [comment]
I'll open the PR again, I somehow messed it up on my end. Sorry for that.

**Pull request closed without merging**